filament/include/filament/ColorGrading.h

```diff
@@ -53,6 +53,9 @@ class UTILS_PUBLIC ColorGrading : public FilamentAPI {
 
         Builder& toneMapping(ToneMapping toneMapping) noexcept;
 
+        // TODO: document: temperature from -1 to +1, tint from -1 to +1; clipped otherwise
```

Comment:
Shouldn't this be a value in K? or is there a way to go from either a value in K or from a standard white point (e.g. D65) to this -1/+1 value?

Reply:
White balance can be applied in different way. It's common to treat color/white balance as an adjustment between -100 and +100, both for the temperature and the tint. Basically think of it as an offset from D65 both horizontally (temperature) and vertically (tint).

Reply:
I imagine this depends from the color temperature of the screen.  I'm wondering if we shouldn't/couldn't use this to correct for screens that are too blue "because it looks better".

Shouldn't we have a utility then, that we could give it the assumed white-point (e.g. D65) and the desired one, and it would return the proper value for this function?

What does -100 or +100 mean? do they correspond to a known temperature? I think we should have more comments about what the parameters are and how to use them.

Reply:
Yeah I still need to write the documentation (and Java bindings). Remember the main use case for color grading is artistic control. This I not to fix broken displays (because you'd need to know what the display's true white point is as well). As for setting the temperature directly (like you'd do in Lightroom) it works only if you know the white point of the original image which you do get from a camera, but in our case we don't know what it is. We must also not confuse the white point of the color space (always D65 for us) and the white point of the image.